Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/DocumentSchema.h

```diff
@@ -265,7 +265,8 @@ namespace AZ::DocumentPropertyEditor
         {
             Dom::Value result(Dom::Type::Object);
             result[EntryDescriptionKey] = Dom::Value(attribute.m_description, true);
-            result[EntryValueKey] = Dom::Value(attribute.m_value);
+            result[EntryValueKey] = Dom::Value();
+            result[EntryValueKey].SetUint64(attribute.m_value);
```

Comment:
I made this change so the Dom::Value ctor is no longer called. That "fixed" the ambiguity issue in Linux builds. We still need to figure out what caused it and find a resolution.

Merging as-is for now with a note to revisit this with lumberyard-employee-dm after the break.